### ISSUE TITLE
Start/Finished Dates

### ISSUE THREAD

**Issue by JBaby9783** [advplyr/audiobookshelf#2630]
### Describe the feature/enhancement

I'm adding my Audible library to ABS and I'm marking the books I've read as finished as I add them. I know all of the start/finished dates for all of these books. Can we please have the option to edit these dates? Thanks!

**Comment by pwinnski** [comment]
This is already possible by editing the database file, but presumably you would like to be able to do in the ABS UI. Just noting this for anyone willing to edit DB files!

**Comment by JBaby9783** [comment]
> This is already possible by editing the database file, but presumably you would like to be able to do in the ABS UI. Just noting this for anyone willing to edit DB files!

This is true. However, that's not good enough. Users don't have access to the DB.

**Comment by SawkeeReemo** [comment]
I second this request. Please and thank you!

**Comment by nivek1385** [comment]
Glad that I searched before adding a feature request. I'm also looking to do this so that it doesn't affect my Year in Review.

**Comment by jerryhcooke** [comment]
I also support this, since while you can edit the database, the section of the database that appears to hold the date finished doesn't have the title of the book, just an ID, which makes modifying it a tedious act of looking up IDs and matching.

**Comment by Vito0912** [comment]
Fyi, I made a tool upon request a while back to update the finished date:  
<https://abstoolbox.vito0912.de/tool/listen-date-updater>
 
Sadly, ABS does not support changing the start date once it has been set, so this is not an option.

**Comment by jerryhcooke** [comment]
@Vito0912 thank you, that's super useful!

**Comment by ZLoth** [comment]
https://github.com/Vito0912/absToolbox/issues/7
https://github.com/advplyr/audiobookshelf/issues/4915

**Comment by luiscaiano** [comment]
i also noted, that using Api
PATCH /api/me/progress/<libraryItemId>
does not work, startedAt is ignored, i deleted the progress item, and tried, will multiple combinations, at least expecting that the start time, could match the end date, but no luck
so now, i have a bunch of books, that i started after i read them,

**Comment by Vito0912** [comment]
You cannot change a started at after progress was created. You can freely while creating.
Imho it makes sense, but you can easily circumvent that by deleting the progress before. That's also how I do it with the toolbox linked above.
